tests/math/test_functions.py

```diff
@@ -2478,3 +2478,93 @@ def test_jax_ndim():
         _ = qml.math.ndim(jax.numpy.array(3))
 
     mock_ndim.assert_called_once_with(3)
+
+
+class TestSetIndex:
+    @pytest.mark.torch

```

Comment:
The file is marked `pytestmark = pytest.mark.all_interfaces` therefore you should remove those markers.